mobile/test_driver/app_test.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("CRUD class", () {
+    FlutterDriver driver;
+
+    setUpAll(() async {
+      driver = await FlutterDriver.connect();
+    });
+
+    tearDownAll(() async {
+      if (driver != null) {
+        driver.close();
+      }
+    });
+
+    test("Login an account properly validated", () async {
```

Comment:
Where are the edge cases? We need tests to make sure things are properly working